mappings/net/minecraft/world/World.mapping

```diff
@@ -278,9 +281,13 @@ CLASS none/aiw net/minecraft/world/World
 		ARG 0 value
 	METHOD j getRainGradient (F)F
 		ARG 0 deltaTicks
+	METHOD j getSkyLightLevel (Lnone/cn;)I
```

Comment:
Is there a method that specifically checks only for block light, too? I hope it's not disguised as the other `getLightLevel`.